src/main/java/edition/academy/seventh/serivce/BookService.java

```diff
@@ -2,28 +2,38 @@
 
 import edition.academy.seventh.database.model.Book;
 import edition.academy.seventh.repository.BookRepository;
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-
-import java.util.List;
-
-
 /**
  * Service that provides API responsible for adding books into database.
  *
  * @author Agnieszka Trzewik
  */
 @Service
-public class BooksService {
-
+public class BookService {
   private BookRepository bookRepository;
 
   @Autowired
-  public BooksService(BookRepository bookRepository) {
+  public BookService(BookRepository bookRepository) {
     this.bookRepository = bookRepository;
   }
 
+  /**
+   * Add books to database.
+   *
+   * @param books books
+   */
   public void addBooksToDataBase(List<Book> books) {
     bookRepository.addBooksToDataBase(books);
   }
+
+  /**
+   * Get books entities from database and pack them into List<Book>
```

Comment:
Ludki, angielski halo. Po co dokumentacja, która nie jest pisana po angielsku. Jak ktoś nie jest pewny jak złożyć jakieś zdanie to dać mi znać plox

Retrieves **book** entities from database and pack**s** them into List